api_devauth.go

```diff
@@ -254,11 +254,7 @@ func (d *DevAuthHandler) UpdateDeviceStatusHandler(w rest.ResponseWriter, r *res
 		return
 	}
 
-	devurl := utils.BuildURL(r, uriDevice, map[string]string{
-		":id": devid,
-	})
-
-	w.Header().Add("Location", devurl.String())
+	w.Header().Add("Location", "../../devices/"+devid)
```

Comment:
The endpoint is `/api/0.1.0/devices/<id>/status`, so the URL can be either `../<id>` or `../../devices/<id>`. Not sure if there's any preference here though.